itests/org.openhab.binding.mqtt.homie.tests/src/main/java/org/openhab/binding/mqtt/HomieImplementationTest.java

```diff
@@ -72,6 +71,7 @@
 
     private MqttService mqttService;
     private MqttBrokerConnection embeddedConnection;
+    private MqttBrokerConnection publishConnection;
     private MqttBrokerConnection connection;
```

Comment:
I think I'm confused here. Why do we need 3 connections? One for publishing, one for subscribing, but a third?

Reply:
We can remove that again. I was originally thinking that it was problem with the embeddedConnection and added that when I tested with an external broker.

Look at my latest commit: I think all tests should succeed, but both with wildcards fail, the others succeed. So it seem to be a moquette problem.